src/Analyzers/CSharp/Analyzers/SimplifyInterpolation/CSharpHelpers.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.SimplifyInterpolation;
+
+namespace Microsoft.CodeAnalysis.CSharp.Analyzers.SimplifyInterpolation
+{
+    internal sealed class CSharpHelpers : AbstractHelpers
```

Comment:
either make these all nested types, or change the names to SimplifyInterpolationHelpers :)  having these vague names is not fun when doing 'nav to' :)